lib/jekyll/liquid_renderer.rb

```diff
@@ -53,7 +53,7 @@ def self.format_error(e, path)
     private
 
     def filename_regex
-      %r!\A(#{source_dir}/|#{theme_dir}/|\W*)(.*)!oi
+      @filename_regex ||= %r!\A(#{source_dir}/|#{theme_dir}/|\W*)(.*)!i
```

Comment:
Just curious: Why did you get rid of the `o` modifier?

Reply:
The `o` modifier stands for "interpolate once" which I feel is *unnecessary" when the regex itself is going to be created once per `Site` instance since the entire method has now been memoized..